src/containers/Tenant/Diagnostics/DiagnosticsPages.ts

```diff
@@ -90,7 +90,7 @@ export const DATABASE_PAGES = [
 ];
 
 export const TABLE_PAGES = [overview, schema, topShards, nodes, graph, tablets, hotKeys, describe];
-export const COLUMN_TABLE_PAGES = [overview, schema, topShards, nodes, graph, tablets, describe];
+export const COLUMN_TABLE_PAGES = [overview, schema, topShards, nodes, tablets, describe];
```

Comment:
`Graph` tab requires `TablePartitions`, `TablePartitionStats` and `TablePartitionMetrics` fields from table `/describe`. There is no such fields for column tables, for them this tab will be always empty